easybuild/easyconfigs/x/Xvfb/Xvfb-21.1.6-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,123 @@
+easyblock = 'Bundle'
+
+name = 'Xvfb'
+version = '21.1.6'
```

Comment:
There is a 21.1.8 available (https://www.x.org/releases/individual/xserver/).

Reply:
Thanks for the review. I will add more recent versions.